corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -688,6 +688,27 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
     });
 
+    var CaseDetailFooterView = Marionette.View.extend({
+        tagName: "div",
+        className: "",
+        events: {
+            "click": "tabClick",
+        },
+        getTemplate: function () { // either return the multi-select footer or the regular case detail footer or empty footer depending on isMultiSelect and IsPersistant detail
+            var id = "#module-case-detail";
+            if (this.isPersistentDetail) {
+                id = "#module-case-detail-persistent-detail";
```

Comment:
I was a little confused by the empty template. Do you think it'd be better/worse if this line just did a `return "";`?

Reply:
very fair- switched with [c92c0f4](https://github.com/dimagi/commcare-hq/pull/31349/commits/c92c0f471a5bafaa400ceb296dbd5974873132fc)